garrysmod/lua/vgui/dtextentry.lua

```diff
@@ -310,6 +329,12 @@ function PANEL:SetValue( strValue )
 
 end
 
+function PANEL:SetPlaceholderText( strValue )
+	
+	self.m_sPlaceholderText = tostring( strValue )

```

Comment:
Is that necessary? Why should it have a different behaviour to `Panel:SetText(...)` and `DTextEntry:SetValue`?